packages/dara-components/js/smart/chat/chat.tsx

```diff
@@ -51,6 +68,9 @@ const StyledChat = injectCss(UiChat);
 function Chat(props: ChatProps): JSX.Element {
     const [style, css] = useComponentStyles(props);
     const [value, setValue] = useVariable(props.value);
+    const anonymousUser: UserData = { identity_name: 'Anonymous' };
```

Comment:
this is sound better to me than throwing an error so feel free to keep it like this :)